docker-compose.yaml

```diff
@@ -117,7 +117,7 @@ services:
     build:
       dockerfile: docker/Dockerfile.backend-worker
       context: .
-    command: celery -A oasst_backend.celery_worker worker -l info
+    command: celery -A oasst_backend.celery_worker worker -l info -E
```

Comment:
What is the `-E` flag for here?

Reply:
The -E flag hel to list all the task in the worker